Formula/wine.rb

```diff
@@ -35,16 +35,9 @@ class Wine < Formula
   end
 
   devel do
-    url "https://dl.winehq.org/wine/source/2.x/wine-2.13.tar.xz"
-    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-2.13.tar.xz"
-    sha256 "6fbca3ddd19b061ce65a267639efae93cf38cfabbe4a9c148a17a44703587ef8"
-
-    # Patch to fix screen-flickering issues. Still relevant on 2.3.
```

Comment:
patch url is `https://raw.githubusercontent.com/Homebrew/formula-patches/74c2566/wine/2.14.patch`